manifests/mod/watchdog.pp

```diff
@@ -10,7 +10,11 @@
 ) {
   include apache
 
-  apache::mod { 'watchdog':
+  $module_builtin = facts['os']['family'] in ['Debian']
+
+  if $module_builtin {
+    apache::mod { 'watchdog':
+    }
```

Comment:
Interesting! Tests FTW 😃  

Out of curiosity, what other platforms did you run the tests against? I can run this against [our internal infrastructure](https://github.com/puppetlabs/puppetlabs-apache/blob/bd7942973532003ee00c8a3e32fc4399ef75ee28/provision.yaml#L43-L100) to get more coverage if you felt it was valuable?

Reply:
I proceeded this way:

1. add an acceptance test, commit and push, see where it fails
2. inspect failures and fix them appropriately when it's not supposed to fail (RedHat 8: add a dependency on the `mod_md` package; Debian 10 / Ubuntu 20: handle the built-in module);
3. filter all failures which are expected because the version of apache is too old or no package is availalbe using the `@note Unsupported platforms` stanza.

It seems that all plaforms listed in your link should either work or be excluded, so running against your internal infrastructure should pass :wink:

Reply:
Kindly reminder for @sanfrancrisko in case my last message slept through :wave:.

Reply:
Apologies for the delay getting back @smortex - there was a mix up on covering this module last week. I'll look to get this over the line and released this week.